src/main/java/com/google/api/generator/gapic/composer/grpc/GrpcServiceCallableFactoryClassComposer.java

```diff
@@ -0,0 +1,204 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.composer.grpc;
+
+import com.google.api.gax.grpc.GrpcCallSettings;
+import com.google.api.gax.grpc.GrpcCallableFactory;
+import com.google.api.gax.grpc.GrpcStubCallableFactory;
+import com.google.api.generator.engine.ast.MethodDefinition;
+import com.google.api.generator.engine.ast.TypeNode;
+import com.google.api.generator.gapic.composer.common.AbstractServiceCallableFactoryClassComposer;
+import com.google.api.generator.gapic.composer.comment.StubCommentComposer;
+import com.google.api.generator.gapic.composer.store.TypeStore;
+import com.google.api.generator.gapic.composer.utils.ClassNames;
+import com.google.longrunning.Operation;
+import com.google.longrunning.stub.OperationsStub;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class GrpcServiceCallableFactoryClassComposer
+    extends AbstractServiceCallableFactoryClassComposer {
+  private static final GrpcServiceCallableFactoryClassComposer INSTANCE =
+      new GrpcServiceCallableFactoryClassComposer();
+
+  protected GrpcServiceCallableFactoryClassComposer() {
+    super(
+        createTypes(),
+        new StubCommentComposer("gRPC"),
```

Comment:
I do have Transport enum in the next PR, I just removed all that kind of stuff to make the nex PR complete one (i.e. transport abstraction stuff is introduced there, not partially here). For this specific case (comment composer) it is literally about which string should be shown to the user (because even in the enum it would be called `GRPC`, not `gRPC`, so even with enum it is not quite the same; at the same time it is  just about posting a comment, so I felt like it did not worth some fancier solution than this).

For the second part of your comment I'm not sure I understand what you mean by transport-specific state redefined in abstract classes. All that state is immutable, it is basically just a config. Transport-specific things are different for each composer (there is some intersection, but not that much). If she move  every transport-specific stuff into context it would mean we would have to populate the context with a lot of stuff which makes sense only for one composer (i.e. the context becomes a big pile of things, many of which are not even known on the moment of initialization of the context). 

I'm fine with either approach, but I prefer the current one, as it keeps the transport-specific transport-agnostic business within the composer (does not propagate it into a much more global and shared thing as the Context). I like viewing `AbstractClassComposer+GrpcClassComposer+HttpJsonClassComposer` as a very close family of classes, basically one class split in 3 to take care of a very specific problem. Introducing context would unnecessarily propagate the transport-specific thing to a bigger context (i.e. will make the change architecturally more different from the original implementation).